src/swagger-specs/policy-service.yaml

```diff
@@ -960,124 +955,211 @@ parameters:
+              href:
+                type: string
+                description: The URL template.
                 format: uri
-                example: ./76131228-7654-11e8-adc0-fa7ae01bbebc
+              templated:
+                type: string
+                description: When present and set to `true`, indicates that this is a templated URL.
+  _linksPagingCoreLabels:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/labels/core?{?limit,start,property}
+          templated: true
+  _linksPagingCustomLabels:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/labels/custom?{?limit,start,property}
+          templated: true
+  _linksPagingCorePolicies:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/policies/core?{?limit,start,property}
+          templated: true
+  _linksPagingCustomPolicies:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/policies/custom?{?limit,start,property}
+  _linksPagingCoreMarketingActions:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/core?{?limit,start,property}
+  _linksPagingCustomMarketingActions:
+    allOf:
+      - $ref: '#/definitions/_linksPaging'
+    example:
+      _links:
+        _page:
+          href: https://platform.adobe.io:443/data/foundation/dulepolicy/marketingActions/custom?{?limit,start,property}
   PolicyExpression:
     description: >-
-      A boolean expression to be evaluated over the presence of data usage labels. Note that a PolicyExpression **must** consist of **either** a label, OR an operator and operands. Each operand in turn is itself a PolicyExpression. It is an error to specify both a label and an operator/operand combination (in spite of what the the Swagger UI-generated example may display). See the section on Policy Expressions at the top of this page for full description and example.
+      A boolean expression to be evaluated over the presence of data usage labels, referred to as a "policy expression". Note that a policy expression **must** consist of **either** a label, OR an operator and operands. Each operand in turn is itself a policy expression. You cannot specify both a label and an operator/operand in combination. See the [Policy Service API guide](https://experienceleague.adobe.com/docs/experience-platform/data-governance/api/policies.html?lang=en#create-policy) for more details on how to configure this property.
```

Comment:
I don't think we need the comma in the sentence - so it should just be "... must consist of either a label OR an operator and an... ".